[2025-06-04T07:02:03.475+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: auto_pull_dags_from_github.git_pull_latest_dags scheduled__2025-06-04T06:00:00+00:00 [queued]>
[2025-06-04T07:02:03.482+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: auto_pull_dags_from_github.git_pull_latest_dags scheduled__2025-06-04T06:00:00+00:00 [queued]>
[2025-06-04T07:02:03.483+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-06-04T07:02:03.493+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): git_pull_latest_dags> on 2025-06-04 06:00:00+00:00
[2025-06-04T07:02:03.499+0000] {standard_task_runner.py:60} INFO - Started process 265 to run task
[2025-06-04T07:02:03.503+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'auto_pull_dags_from_github', 'git_pull_latest_dags', 'scheduled__2025-06-04T06:00:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/auto_pull_dags.py', '--cfg-path', '/tmp/tmpojq7_z_4']
[2025-06-04T07:02:03.507+0000] {standard_task_runner.py:88} INFO - Job 42: Subtask git_pull_latest_dags
[2025-06-04T07:02:03.579+0000] {task_command.py:423} INFO - Running <TaskInstance: auto_pull_dags_from_github.git_pull_latest_dags scheduled__2025-06-04T06:00:00+00:00 [running]> on host 036d21481f2d
[2025-06-04T07:02:03.645+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='aswin.b@sganlytics.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='auto_pull_dags_from_github' AIRFLOW_CTX_TASK_ID='git_pull_latest_dags' AIRFLOW_CTX_EXECUTION_DATE='2025-06-04T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-04T06:00:00+00:00'
[2025-06-04T07:02:03.647+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-06-04T07:02:03.648+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dags && git pull origin main']
[2025-06-04T07:02:03.659+0000] {subprocess.py:86} INFO - Output:
[2025-06-04T07:02:03.661+0000] {subprocess.py:93} INFO - /usr/bin/bash: line 1: git: command not found
[2025-06-04T07:02:03.662+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2025-06-04T07:02:03.670+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2025-06-04T07:02:03.673+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=auto_pull_dags_from_github, task_id=git_pull_latest_dags, execution_date=20250604T060000, start_date=20250604T070203, end_date=20250604T070203
[2025-06-04T07:02:03.685+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/email.py:154: RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
  send_mime_email(e_from=mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)

[2025-06-04T07:02:03.686+0000] {email.py:270} INFO - Email alerting: attempt 1
[2025-06-04T07:02:05.954+0000] {email.py:281} INFO - Sent an alert email to ['aswin.b@sganlytics.com']
[2025-06-04T07:02:08.153+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 42 for task git_pull_latest_dags (Bash command failed. The command returned a non-zero exit code 127.; 265)
[2025-06-04T07:02:08.197+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-06-04T07:02:08.229+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
